ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -897,20 +897,19 @@ export const DatasetPage = fp.flow(
       conceptSets: ConceptSet[],
       prepackagedConceptSets: PrePackagedConceptSetEnum[]
     ): DomainWithConceptSetId[] => {
-      return conceptSets
-        .map((cs) => ({
+      return (
+        conceptSets.map((cs) => ({
           conceptSetId: cs.id,
           domain:
             cs.domain === Domain.PHYSICAL_MEASUREMENT
               ? Domain.MEASUREMENT
               : cs.domain,
+        })) as DomainWithConceptSetId[]
```

Comment:
IntelliJ didn't like when I removed the `conceptSetId` from below until I added this `as`